Simplenote/Classes/SPEditorTextView.h

```diff
@@ -16,15 +15,14 @@
 
 @interface SPEditorTextView : SPTextView
 
-@property (nonatomic) BOOL lockTagEditorPosition;
-
-@property (nonatomic, strong) SPTagView *tagView;
 @property (nonatomic, strong) UIFont *checklistsFont;
 @property (nonatomic, strong) UIColor *checklistsTintColor;
 
 @property (nonatomic, readonly) BOOL isInserting;
 @property (nonatomic, readonly) BOOL isDeletingBackward;
 
+@property (nonatomic, strong) void (^onContentPositionChange)(void);
```

Comment:
Trying to remember, I think this should be `copy` rather than `strong` ? In old ObjC jargon?

Reply:
At first it blocks had to be declared as `copy`, but with ARC and modern `clang` `strong` is just fine :-)